web/src/components/FeedbackToolbarItem.tsx

```diff
@@ -36,7 +36,11 @@ const FeedbackToolbarItem = ({
   return (
     <Tooltip text={viewportSmall ? null : dataTip} flow='up' mediumViewportFlow='right' smallViewportFlow='down'>
       <StyledFeedbackToolbarItem className={className} onClick={handleLinkClick} aria-label={dataTip}>
-        <FontAwesomeIcon className={className} icon={isPositiveRatingLink ? faSmile : faFrown} />
+        <FontAwesomeIcon
+          className={className}
+          icon={isPositiveRatingLink ? faSmile : faFrown}
+          title={isPositiveRatingLink ? 'faSmile' : 'faFrown'}
```

Comment:
Title should not be used for this purpose, it is shown as a tooltip:
![image](https://user-images.githubusercontent.com/29095487/181003060-98fa4d66-d2c4-4a26-af03-ec9454cfa452.png)
I think you could just remove the tests, they don't really test important stuff.